katas/langtonant/matias/main.js

```diff
@@ -0,0 +1,38 @@
+class Ant {
```

Comment:
Looks like implementation still in progress, please let me know if you are planning to push more changes to complete the Kata